Framework/PythonInterface/test/python/plugins/algorithms/MaskBTPTest.py

```diff
@@ -217,31 +217,71 @@ def test_components(self):
         self.assertEqual(10 * 90, len(masked))
 
     def test_d33(self):
-        ws = LoadEmptyInstrument(InstrumentName='D33', OutputWorkspace="D33")
+        ws = LoadEmptyInstrument(InstrumentName='D33')
         mask_rear = MaskBTP(Workspace=ws, Components='back_detector')
         mask_front = MaskBTP(Workspace=ws, Bank="1-4", Tube="15", Pixel='0-10')
 
         self.assertEqual(len(mask_rear), 256 * 128)
         self.assertEqual(len(mask_front), 4 * 11)
+        global CHECK_CONSISTENCY
```

Comment:
Maybe I wasn't very clear, this is not what I meant.
Let the ILL tests be similar to the others.
That is run self.checkConsistentMask, but do not alter the global flag.
Let it be off, as the comment says it makes the test very slow.
The purpose of it existing, is to enable this check by modifying the global flag and not touching the tests themselves.

Reply:
Then, locally, you switch on, make sure it passes, but to push to the branch, switch if back off.

Reply:
My bad, it makes far more sense this way